README.md

```diff
@@ -112,6 +112,24 @@ curl -sL https://github.com/binary-install/binstaller/releases/latest/download/i
      sh "$tmpfile" -b "$INSTALL_DIR"; rm -f "$tmpfile")
 ```
 
+#### Test Installation (Dry Run)
+
+Test the installer without actually downloading or installing anything:
+
+```bash
+# Test the installer with dry run mode
+curl -sL https://github.com/binary-install/binstaller/releases/latest/download/install.sh | \
+    (tmpfile=$(mktemp); cat > "$tmpfile"; \
+     gh attestation verify --repo=binary-install/binstaller --signer-workflow='actionutils/trusted-go-releaser/.github/workflows/trusted-release-workflow.yml' "$tmpfile" && \
```

Comment:
I don't expect users use dry-run mode with this kind of download->verify->run one liner usage.
It's more like for validation and debugging purpose when preparing a config and an installer.

Instead of writing a section in installation section, we should mention the installer dry-run mode briefly and concisely with `binst check` section which is also related to validating installer config.